src/foam/nanos/crunch/connection/CapabilityPayloadDAO.js

```diff
@@ -228,6 +229,10 @@ foam.CLASS({
     {
       name: 'put_',
       javaCode: `
+      var pm = new PM(CapabilityPayloadDAO.getOwnClassInfo().getId(), "capabilityPayloads.put");
```

Comment:
With regard to naming, in this case the first part is already CapabilityPayloadDAO, so the second 'capabilityPayloads.put can be just 'put'